Documentation/General/BuildSystem/HowToBuildMacOS.md

```diff
@@ -0,0 +1,26 @@
+How to build for MacOS {#HowToBuildMacOS}
```

Comment:
Add a page to the Wiki:
https://github.com/ezEngine/ezEngine/wiki/Building-ezEngine